Домашнее задание №1/Number 2/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Number 2</title>
+    <script src="Figure.js"></script>
+    <script >
+
+        window.onload = () => {
+
+            //Функция подсчета площади
+            function calculateArea(figure, parametr1, parametr2) {
+
+                let area;
+                let input = [];
+                input.push(parametr1);
+
+                if (figure === 'square' || figure === 'Square') {
+                    area = parametr1*parametr1;
+                } else if (figure === 'circle' || figure === 'Circle') {
+                    area = Math.floor(parametr1*parametr1*Math.PI);
+                } else if (figure === 'triangle' || figure === 'Triangle'){
+                    area = (parametr1*parametr2)/2;
+                    input.push(parametr2);
+                }
+
+                return new Figure(figure, area, input);
+
+            }
+
+            //Получение объекта
+            let result = calculateArea('square', 3);
+            console.log(result);
+
+            //Реструктуризация(практика)
+            let {name: figure, area, input: [parametr1]} = result;
```

Comment:
```suggestion
            let {figure, area, input: [parametr1]} = result;
```
чтобы код работал с предложенным ранее вариантом реализации:)